ember-server/src/main/scala/org/http4s/ember/server/internal/ServerHelpers.scala

```diff
@@ -232,7 +228,7 @@ private[server] object ServerHelpers {
                 resp.headers.get(Connection).exists(_.hasClose)
             )
         }
-        .drain ++ Stream.eval(socket.close).drain
```

Comment:
I think we can actually get rid of this on `series/0.21` and `series/0.22` as well? The socket resource is lifted into the stream on a new scope, so it should be closed at the end here anyways. Maybe we just wanted to be explicit here?

Reply:
I think you're correct.  I only noticed it here because it disappeared in fs2-3.